### ISSUE TITLE
awswrangler.s3.to_parquet set table properties when creating a table dynamically

### ISSUE THREAD

**Issue by fjbecerra** [awslabs/aws-data-wrangler#985]
hey,
Im creating tables dynamically with `awswrangler.s3.to_parquet`, but I'd also like to set table properties, similar to TBL_PROPERTIES mentioned here https://docs.aws.amazon.com/athena/latest/ug/create-table.html

I don't see anything like that in the wrangler doc. https://aws-data-wrangler.readthedocs.io/en/2.12.1/stubs/awswrangler.s3.to_parquet.html#awswrangler.s3.to_parquet

Is it possible to do this?

**Comment by jaidisido** [comment]
`s3.to_parquet` already has a `database` and `table` input parameters which enable creating a Glue table based on the S3 files created. The table would be available in Athena too since it looks at the same Glue catalog

**Comment by fjbecerra** [comment]
correct, however the table is created with default properties, right?, if I wanted to add or update the table properties, does the current `to_parquet` support it in any of its arguments ?

**Comment by jaidisido** [comment]
The schema/partitions portion of the table properties is inferred from the input Pandas data frame, for any other property of the table you can leverage `wr.catalog` methods, for example:
- [create_parquet_table](https://aws-data-wrangler.readthedocs.io/en/stable/stubs/awswrangler.catalog.create_parquet_table.html#awswrangler.catalog.create_parquet_table)
- [overwrite_table_parameters](https://aws-data-wrangler.readthedocs.io/en/stable/stubs/awswrangler.catalog.overwrite_table_parameters.html#awswrangler.catalog.overwrite_table_parameters)
